pkg/controller/controller_test.go

```diff
@@ -3653,7 +3653,7 @@ var _ = Describe("CSIControllerService", func() {
 					Return(gopowerstore.Cluster{Name: validClusterName}, nil)
```

Comment:
Copyright year is:  © 2021-2022. Can you please change it to Copyright © 2021-2023?